File: luiz020/fluxo/Q8.py
k1 = int(input("Digite o primeiro número: "))
k2 = int(input("Digite o segundo número: "))
k3 = int(input("Digite o terceiro número: "))

if k1 < k2 and k2 < k3:
    if k1 < k2:
        print("Não está em Ordem Decrescente")
    else:
        print("Está em Ordem Decrescente")
        if k2 < k3:
            print("Não está em Ordem Decrescente")
        else:
            print("Está em Ordem Decrescente")
else:
    print("Está em Ordem Decrescente")

File: maria018/lista2_poo/q8.java
import java.util.Scanner;
public class q8 {
    public static void main (String[] args) {
    
        Scanner s;
        int n1, n2, n3;
        s = new Scanner (System.in);

        System.out.print("Digite o primeiro numero: " );
        n1 = s.nextInt();
        
        System.out.print("Digite o segundo numero: ");
        n2 = s.nextInt();
    
        System.out.print("Digite o terceiro numero:");
        n3 = s.nextInt();
    
        if ((n1 < n2) && (n1 < n3)) {
            System.out.println("O menor eh o " + n1);
        }
        if ((n2 < n1) && (n2 < n3)) {
            System.out.println("O menor eh o " + n2);
        }   
        if ((n3 < n1) && (n3 < n2)) {
            System.out.println("O menor eh o " + n3); 

       }
    }
} 
